lib/modules/manager/buildpacks/extract.ts

```diff
@@ -0,0 +1,92 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { getDep } from '../dockerfile/extract';
+import type {
+  ExtractConfig,
+  PackageDependency,
+  PackageFileContent,
+} from '../types';
+import { type ProjectDescriptor, ProjectDescriptorToml } from './schema';
+
+const dockerPrefix = /^docker:\/\//;
+
+function parseProjectToml(content: string): ProjectDescriptor | null {
+  let res = ProjectDescriptorToml.safeParse(content);
+
+  if (res.success) {
+    return res.data;
+  }
+
+  return null;
+}
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string,
+  config: ExtractConfig,
+): PackageFileContent | null {
+  const deps: PackageDependency[] = [];
+  try {
+    const descriptor = parseProjectToml(content);
+    if (!descriptor) {
+      return null;
+    }
+
+    if (
+      descriptor.io.buildpacks?.builder &&
+      !descriptor.io.buildpacks.builder.startsWith('file://')
```

Comment:
```suggestion
      descriptor.io.buildpacks.builder.startsWith(dockerPrefix)
```

otherwise it will do wron things when they add new prefixes